### PR TITLE
[PW-7576] Add support for query parameters

### PR DESCRIPTION
**Pull request by antolo-arch**
**Description**
Added support for query parameters, added an example in the readme

**Tested scenarios**
Tested whether the query parameters are being passed in the method, tested the creation of the query string

### CODE REVIEW

**Review by jillingk — DISMISSED** [review]

File: .github/workflows/unittest.yml [hunk 1]
```diff
@@ -21,6 +21,7 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           pip install tox
+          sudo apt-get update
```
Comment (AlexandrosMor) [inline .github/workflows/unittest.yml:24]: Should we update every time that it runs ?

File: Adyen/util.py [hunk 2]
```diff
@@ -98,3 +98,7 @@ def is_valid_hmac_notification(dict_object, hmac_key):
             merchant_sign = generate_notification_sig(dict_object, hmac_key)
             merchant_sign_str = merchant_sign.decode("utf-8")
             return hmac.compare_digest(merchant_sign_str, expected_sign)
+
+
+def get_query(query_parameters):
+    return '?' + '&'.join([key + '=' + str(query_parameters[key]) for key in query_parameters])
```
Comment (AlexandrosMor) [inline Adyen/util.py:104]: Should we use string formatting ? For example 
return "?{}".format(
    "&".join("{}={}".format(k, v) for k, v in query_parameters)
)

File: .github/workflows/unittest.yml (hunk 1 above)
Comment (jillingk) [reply .github/workflows/unittest.yml:24]: https://docs.github.com/en/actions/using-github-hosted-runners/customizing-github-hosted-runners#installing-software-on-ubuntu-runners
Github recommends this as well

File: Adyen/util.py (hunk 2 above)
Comment (antolo-arch) [reply Adyen/util.py]: how about this one?
`
'?' + '&'.join(["{}={}".format(k, v) for k, v in query_parameters.items()])`

Comment (AlexandrosMor) [reply Adyen/util.py]: yes this one looks ok

File: .github/workflows/unittest.yml (hunk 1 above)
Comment (AlexandrosMor) [reply .github/workflows/unittest.yml:24]: Thank you, good to know

**Review by AlexandrosMor — APPROVED** [review]

**Pull request merged**